docs/content/fin/fin-help.md

```diff
@@ -363,7 +365,7 @@ aliases:
 	
 	View output from containers.
 	
-	Usage: logs [options] [SERVICE...]
+	Usage: logs [options] [--] [SERVICE...]
```

Comment:
Does the addition of `[--]` have an impact on this feature?

Reply:
I ran  `./scripts/fin-help-generate.sh` to update `fin help` docs. 

This particular output comes from `docker-compose logs --help`, which we do not control. docker-compose version was updates, thus the change.